.gitignore

```diff
@@ -80,3 +80,29 @@ databox-export-service/
 databox-app-twitter-sentiment/
 
 databox-driver-twitter-stream/
+
+core-arbiter/
+
+core-container-manager/
+
+core-export-service/
```

Comment:
If I were to start getting really picky about naming, perhaps `core-exporter` would fit better with the other core libs? (But really not important or even convincing :)